packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
nit: b/c naming is always a pain.  Should this be `forceRenameUiComponent`?  I could see looking for where we rename aura components and discounting this file b/c it's explicitly calls out LightningComponent. 

If we want to keep it I could also see adding a JSDoc to the executor to document that this is for both LWC and Aura components.

Reply:
@gbockus-sf  I think "Lightning Component" includes both Aura and LWC according to this [Salesforce Lightning Component Library page](https://developer.salesforce.com/docs/component-library/overview/components).